config.py

```diff
@@ -163,18 +163,18 @@
 fs = os.path.sep
 root = os.path.dirname(os.path.abspath(__file__)) + fs + ".." + fs
 region = param["region"]
-year = param["year"]
+year = str(param["year"])
```

Comment:
This might create conflicts with other functions

Reply:
how did we use to enter the year in the paths before? it has to be a string